.storybook/lucid-docs-addon/ExampleCode.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import SyntaxHighlighter, {
+	registerLanguage,
+} from 'react-syntax-highlighter/prism-light';
+import jsx from 'react-syntax-highlighter/languages/prism/jsx';
+import okaidia from 'react-syntax-highlighter/styles/prism/okaidia';
+
+const okaidiaTabSpace2 = {
+	...okaidia,
+	'code[class*="language-"]': {
+		...okaidia['code[class*="language-"]'],
+		tabSize: '2',
+	},
+};
+
+registerLanguage('jsx', jsx);
+
+class CodepenDefine extends React.Component {
+	constructor(...args) {
+		super(...args);
+		this.handleCodepen = this.handleCodepen.bind(this);
+	}
+
+	handleCodepen() {
+		const formDOMNode = ReactDOM.findDOMNode(this.form);
+		if (formDOMNode) {
+			formDOMNode.submit();
+		}
+	}
+
+	render() {
+		return (
+			<div onClick={this.handleCodepen}>
+				<form
+					ref={ref => {
+						this.form = ref;
+					}}
+					style={{ display: 'none' }}
+					action="https://codepen.io/pen/define"
+					method="POST"
+					target="_blank"
+				>
+					<input
+						type="hidden"
+						name="data"
+						value={JSON.stringify(this.props.data)}
+					/>
+				</form>
+				{this.props.children}
+			</div>
+		);
+	}
+}
+
+const ExampleCode = ({ code, hasCodepen, packageJson }) => (
+	<section
+		style={{
+			height: '100%',
+		}}
+	>
+		<div style={{ height: '100%' }}>
+			{hasCodepen && (
+				<CodepenDefine
+					data={{
+						title: 'Lucid Example',
+						description: 'This is code from example.',
+						layout: 'right',
+						editors: '0010',
+						html: '<div id="react-root"></div>',
+						js:
+							code
+								.replace(/\s*import _ from 'lodash';\s*/, '')
```

Comment:
could this miss cases like `import { reduce } from 'lodash';`?

Reply:
yes it will miss that. there's a few ways we can make it better:
- parse the example code with babylon
- use more regexes for more cases
- limit code munging by defining webpack resolvers

Reply:
yeah, i vote for webpack resolvers. It'll make the actual example code more idiomatic/copy-pastable as a bonus.